free-1.md

```diff
@@ -1,68 +1,170 @@
 # Free project 1
 
+
+
+An RPG mapmaker with mixed tile sizes and shapes. It would let users create or upload tiles of certain sizes, then specify on the map which patterns belong where by clicking and dragging map regions and selecing the tile to fill the area. There would also be an option to designate what a "border" to the tile looks like, and then the area would have the border on the outside. Tiles that don't fill a square could also have transparency and be layered over others, so borders wouldn't have to be sharp and linear. Once the areas and patterns are specified, an image would be produced. "Objects" could also be placed on the scene, layered above tiled areas.
+
+
+
 ## The user and a language
+
 This section describes who the project would serve and why a language might be a
+
 good way to meet their needs.
 
 
+
+The user would be game developers (or perhaps bored artists). Unfortunately, this feels more like an application than a language, but map generation is a very useful tool.
+
+
+
 ### What's the need?
+
 _What need is met by your idea? Who are you helping? What is that person's
+
 experience like now? What would their experience be like if you could help 
+
 them?_
 
 
+
+Well, existing map generators that I'm aware of generally have set tile sizes, which limits the ability to quickly design maps that don't look as blocky or aren't constrained by having every piece of the map fill a tile of the same size. But to have complete control over a map would require illustrating it entirely from scratch, which would take too long. This would be a tool that might keep the process quick while allowing for more artistic design decisions.
```

Comment:
Mmm, I imagine that allowing for more arbitrarily shaped tiles would also create a need for a way to better describe how to arrange them - what an opportunity!